scan_promela/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "main"
```

Comment:
Package should be called `scan_promela`. Also, the `main` folder is unnecessary: please just move the content to `scan_promela/src/`